restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/proxy/ProxyManagerTest.java

```diff
@@ -20,70 +29,140 @@
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.jboss.shrinkwrap.resolver.api.maven.archive.ShrinkWrapMaven;
 import org.junit.After;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.restcomm.connect.commons.Version;
+import org.restcomm.connect.testsuite.http.RestcommCallsTool;
 //import org.restcomm.connect.telephony.Version;
 
 @RunWith(Arquillian.class)
-@Ignore
 public final class ProxyManagerTest {
     private static final String version = Version.getVersion();
 
     @ArquillianResource
     private Deployer deployer;
 
-    private static SipStackTool tool;
-    private SipStack receiver;
-    private SipPhone phone;
+    final String deploymentUrl = "http://127.0.0.1:8080/restcomm/";
```

Comment:
maybe is a good time to refactor thsi test to support parallelization, see other test to check how it goes

Reply:
updated to parallel test